database/migrations/2024_07_13_113218_remove_username_from_github_repos_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class RemoveUsernameFromGithubReposTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('github_repos', function (Blueprint $table) {
+            $table->dropColumn('username');
```

Comment:
why do you remove `username` from repo?
it's crucial to have this in the table

Reply:
First, I added the username column, but it's not necessary because there is already a userid, so I removed the column. Additionally, there is no username column in the migrations tables of the original github repo.